src/poetry_plugin_pypi_proxy/plugin.py

```diff
@@ -66,7 +66,8 @@ def activate(self, poetry: Poetry, io: IO) -> None:
             return
 
         # Parse the proper for PIP_INDEX_URL but not for publishing.
-        proxy_url = parse_url(proxy_url)
+        parsed_url = generate_poetry_source_config(proxy_url)
```

Comment:
could this be renamed to `source_config = ...`? Think it makes more sense in this context.

Reply:
Good point. Yeah I can change that name.